README.md

```diff
@@ -203,6 +211,16 @@ If a `telemetry` option is provided, ContainerPilot will expose a [Prometheus](h
 
 Details of how to configure the telemetry endpoint and how the telemetry endpoint works can be found in the [telemetry README](https://github.com/joyent/containerpilot/blob/master/telemetry/README.md).
 
+#### Tasks (Optional):
+
+Tasks are commands that are run periodically. They are typically used to perform housekeeping such as incremental back-ups, or pushing metrics to systems that cannot collect metrics through service discovery like Prometheus.
+
+Tasks are defined the following properties:
+
+- `command` is the executable (and its arguments) that will run when the task executes.
+- `frequency` is the time between executions of the task. Supports milliseconds, seconds, minutes. The frequency must be a positive non-zero duration with a time unit suffix. (Example: `60s`) Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
```

Comment:
Hm... is golang's runtime really going to be able to give us precision at this level? Between the userland threads and the stop-the-world GC with a worst-case pause time of 10ms (even in the new go1.5+), I worry that we're going to have folks filing bugs on the tasks feature because of this. Usually you'd use counter or timestamps k/v for the sort of things that require higher precision.

Does this deserve a documentation warning maybe?